pkg/awsclient/client.go

```diff
@@ -113,6 +113,39 @@ type Client interface {
 	DescribeSubnets(*ec2.DescribeSubnetsInput) (*ec2.DescribeSubnetsOutput, error)
 	// CreateTags to apply tags to EC2 resources
 	CreateTags(*ec2.CreateTagsInput) (*ec2.CreateTagsOutput, error)
+
+	// Helper extensions
```

Comment:
Added so that the _helper.go files are considered part of the overall interface so the mocked client will have them as well.